schema/component.schema.json

```diff
@@ -27,6 +27,15 @@
             "translatable": true
           }
         },
+        "ariaQuestion": {
+          "type": "string",
+          "title": "ARIA question",
+          "description": "This will be read out by screen readers instead of reading the 'Display title', 'Body' & 'Instruction' fields when focusing on the options. To be clear and concise, ensure the text encompasses only the question associated.",
```

Comment:
should it say "when focusing on the group or radiogroup." to match what is in the read me instead of "when focusing on the options"